ang/civiawards/reviews-tab/directives/reviews-case-tab-content.directive.js

```diff
@@ -157,7 +159,29 @@
         reset: 1
       });
 
-      loadForm(formUrl);
+      loadForm(formUrl)
+        .on(CRM_FORM_LOAD_EVENT, popupTitleDecodeEntities);
+    }
+
+    /**
+     * Converts HTML entities in popup title to their corresponding characters.
+     */
+    function popupTitleDecodeEntities () {
+      $('.ui-dialog-title').each(function () {
```

Comment:
We need to make sure that we only update the title for the modal form that we opened (multiple modals can be opened at the same time, although that's not very likely to happen in a real scenario).

I think we need to do something like: 

```js
// ...
$formElement = loadForm(formUrl)
        .on(CRM_FORM_LOAD_EVENT, popupTitleDecodeEntities);

// ...
$formElement.find('.ui-dialog-title').each(/* ... */)

```

Reply:
Fixed, thanks.